lib/DDG/Spice/Movie.pm

```diff
@@ -13,13 +13,21 @@ spice alt_to => {
 		# Uses $loc so needs to not cache back end.
 		is_cached => 0,
 		proxy_cache_valid => "200 30d"
+	},
+
+	cast_image => {
+		to => 'https://api.themoviedb.org/3/search/person?query=$1&api_key={{ENV{DDG_SPICE_MOVIEDB_APIKEY}}}',
+		# Uses $loc so needs to not cache back end.
+		is_cached => 0,
+		proxy_cache_valid => "200 30d"
 	}
 };
 
 # This spice will usually be triggered by deep triggers,
 # with a few extra triggers that deep might miss.
-my @triggers = ( 'rotten tomatoes', 'rotten');
+my @triggers = ( 'rotten tomatoes', 'rotten', 'cast', 'casts');
 triggers startend => @triggers;
+triggers start => 'cast of', 'casts of';
```

Comment:
It'd be nice to be able to do `who stars in X`, `who starred in X`, `actors/actresses in X`.